src/WebJobs.Script/Binding/ExtensionBinding.cs

```diff
@@ -22,18 +22,19 @@ namespace Microsoft.Azure.WebJobs.Script.Binding
     public class ExtensionBinding : FunctionBinding, IResultProcessingBinding
     {
         private ScriptBinding _binding;
-        private Collection<Attribute> _attributes;
 
         public ExtensionBinding(ScriptHostConfiguration config, ScriptBinding binding, BindingMetadata metadata) : base(config, metadata, binding.Context.Access)
         {
             _binding = binding;
-            _attributes = _binding.GetAttributes();
+            Attributes = _binding.GetAttributes();
         }
 
+        public Collection<Attribute> Attributes { get; private set; }
```

Comment:
I meant, why is it no longer private?   (internal vs. public doesn't matter here).
I see it's being called from the new FunctionDescriptor.GetTriggerAttributeOrNull()

---
In reply to: [112279468](https://github.com/Azure/azure-webjobs-sdk-script/pull/1433#discussion_r112279468) [](ancestors = 112279468)